test/HyperliquidInterchainToken.js

```diff
@@ -56,18 +60,15 @@ describe('HyperliquidInterchainToken', () => {
     });
 
     describe('Hyperliquid Interchain Token', () => {
-        it('should verify initial deployer state and slot 0 storage', async () => {
-            const slot0 = await provider.getStorageAt(tokenAddress, 0);
-            const deployerFromSlot0 = bytes32ToAddress(slot0);
+        it('should verify initial deployer state', async () => {
+            const deployerSlot = getDeployerStorageSlot();
+            const deployerFromSlot = await provider.getStorageAt(tokenAddress, deployerSlot);
```

Comment:
```suggestion
            const deployerFromSlot = await provider.getStorageAt(tokenAddress, deployerStorageSlot());
```
avoid single-use intermediate variables that only add bloat